hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -262,6 +262,17 @@ protected static function buildFullDetailsQuery($entities, $revision) {
 
       $query['from'][] = $table;
       $query['where'][] = "{$table}.jobcontract_revision_id = " . $revision["{$entity}_revision_id"];
+
+      if ($entity == 'pension') {
+        $query['select'][] = "pension_ov.label as {$entity}__pension_type_label";
+
+        $query['from'][] = 'civicrm_option_group pension_og';
+        $query['from'][] = 'civicrm_option_value pension_ov';
+
+        $query['where'][] = "pension_og.name = 'hrjc_pension_type'";
+        $query['where'][] = 'pension_ov.option_group_id = pension_og.id';
+        $query['where'][] = "pension_ov.value = {$table}.pension_type";
+      }
```

Comment:
Did you run the tests for the hrjobcontract extension? It seems this change is making one of them fail:

![screen shot 2016-12-21 at 08 32 45](https://cloud.githubusercontent.com/assets/388373/21386060/1faaaf42-c758-11e6-9ab6-c54b3adcf832.png)